docs/DeveloperGuide.adoc

```diff
@@ -906,8 +906,12 @@ Use case resumes at step 6.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  Technical requirements: The Addressbook should be able to work on both 32 and 64 bit system
+.  Performance requirements: The Addressbook should be able to give a response within 1 second
+.  Security requirements: The Addressbook should be able to hide personal
```

Comment:
This seems to be more of a use case than an NFR? Or to put it another way, this describes a function (passwords) and is thus not an NFR.